pkg/registry/authentication/rest/storage_authentication.go

```diff
@@ -29,12 +29,14 @@ import (
 	"k8s.io/kubernetes/pkg/registry/authentication/tokenreview"
 )
 
-type RESTStorageProvider struct {
+// StorageProvider is a REST storage provider for storage_authentication
```

Comment:
How about "StorageProvider is a REST storage provider for the authentication API group."